models/jira.yml

```diff
@@ -1,26 +1,27 @@
 version: 2
 
 models:
-  - name: jira__daily_issue_field_history 
+  - name: jira__daily_issue_field_history
     description: >
       Each record represents a snapshot of an issue's selected fields (as specified in
       the `issue_history_columns` variable + sprint and status by default) on a given day.
-      Each issue will have a record for every day in which it was open and/or being updated. 
+      Each issue will have a record for every day in which it was open and/or being updated.
       For currently open issues, the latest date will be the current date, and for closed issues,
       this will be the last time the issue was resolved OR updated in any way (plus a configurable buffer afterward,
       set to a month by default).
       Note: this is an incremental model.
     columns:
+      - name: source_relation
+        description: "{{ doc('source_relation') }}"
       - name: date_day
         description: Date on which the issue had these field values.
       - name: date_week
         description: The start date of the week corresponding to `date_day`.
       - name: issue_id
         description: Foreign key referencing the ID of the `issue` with these values.
       - name: issue_day_id
-        description: Surrogate key hashed on `date_day` and `issue_id`
+        description: Surrogate key hashed on `date_day`, `issue_id`, and `source_relation`
```

Comment:
let's leave the test since it'll still be unique with source_relation in the hash. Could you also note that this surrogate key value is updated in the changelog?